authors/apps/articles/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
Remove this file as it's not needed or used anywhere in the app

Reply:
Removed this too